content/monetization/app-review-process/index.mdx

```diff
@@ -10,30 +10,22 @@ desc: "If you're into building LiveChat integrations, but don't know how to go a
 
 The goal of the LiveChat Marketplace is to provide trusted space for high quality LiveChat apps, plugins, and integrations.
 
-With over 28,000 businesses using LiveChat daily, we want to make sure that what we offer in the Marketplace, provides our customers with tangible value and enriches their LiveChat experience. For that reason, all apps in the LiveChat Marketplace have to meet our criteria for quality and content.
+With over 28,000 businesses using LiveChat daily, we want to make sure that what we offer in the Marketplace, provides our customers with tangible value and enriches their LiveChat experience. For that reason, all apps in the LiveChat Marketplace must meet our criteria for quality and content.
 
-The decision to feature your app on the LiveChat Marketplace depends both on the technical requirements and on our opinion on how your app contributes to the LiveChat ecosystem and community. This document covers most cases, however, some exceptions may be allowed at the reviewer’s discretion.
+The decision to feature your app on the LiveChat Marketplace depends mainly on technical requirements, but not only. We also assess how your app contributes to the LiveChat ecosystem and community. This document covers most cases, however, some exceptions may be allowed at the reviewer’s discretion.
 
-# Developer's responsibilities
-
-As a developer, you need to know how we go about reviewing apps. 
-
-Each app is thoroughly checked before releasing it to the LiveChat Marketplace. It involves the team of three or more people.
-Since we aim to offer top-quality products, we've prepared the list of responsibilites for Developers and App Owners. We highly encourage you to familiarize yourself with all the aspects.
-
-## Quality
 
-Before you submit the app, test it meticulousy to ensure it’s usable and bug-free. Make sure it looks good on desktop, tablet and mobile for the best cross-platform experience.
+# Developer's responsibilities
 
-Keep your code human-readable, clean, and optimized. Respect the generally accepted standards.
-
-Remmeber about the cleanup - remove all unused assets, references, etc.
+As a developer, you need to know how we go about reviewing apps and how to make your app go through the process with success.
 
+Each app is thoroughly checked before releasing it to the LiveChat Marketplace. It involves a team of three or more specialists.
+Since we aim to offer top-quality products, we've prepared the list of responsibilites for developers and app owners. We highly encourage you to familiarize yourself with all the aspects.
```

Comment:
I really think we shouldn't capitalize the roles: "Developer", "App Owners", "Reviewer". It creates unnatural formality making it look like there's a place in Dev Console (or somewhere) where you have to choose who you are (developer, app owner).